core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -83,6 +83,7 @@ private ClusterConfig(Type type, int brokers, int controllers, int disksPerBroke
         this.saslServerProperties = Objects.requireNonNull(saslServerProperties);
         this.saslClientProperties = Objects.requireNonNull(saslClientProperties);
         this.perBrokerOverrideProperties = Objects.requireNonNull(perBrokerOverrideProperties);
+        this.tags = tags;
```

Comment:
`Objects.requireNonNull(tags);`

Reply:
originally I use `Objects.requireNonNull(tags);` but I was not sure if we should make this required since users might not need this, so basically we want/expect users to provide for the display purpose, right?